protostar/self/cache_io.py

```diff
@@ -10,6 +10,10 @@ class CacheIO:
     def __init__(self, project_root_path: Path):
         self._cache_path = project_root_path / Path(self._CACHE_DIR_NAME)
         self._cache_path.mkdir(exist_ok=True)
+        gitignore_path = Path(self._cache_path / ".gitignore")
+        if not gitignore_path.exists():
+            gitignore_path.touch()
```

Comment:
```suggestion
```

nitpick: https://docs.python.org/3/library/pathlib.html#pathlib.Path.write_text write_text creates the file if missing

Reply:
[remove touch](https://github.com/software-mansion/protostar/pull/973/commits/170756a31b730e90fb1707f6a8b6fa48db37d22f)